eta/core/serial.py

```diff
@@ -628,7 +640,7 @@ def serialize(self, reflective=False):
         if reflective:
             d["_CLS"] = self.get_class_name()
             d[self._ELE_CLS_FIELD] = etau.get_class_name(self._ELE_CLS)
-        d[self._ELE_ATTR] = _recurse(self.__elements__, False)
+        d.update(super(Container, self).serialize(reflective=False))
```

Comment:
this fixes a bug that prevented Container attributes other than `self._ELE_ATTR` from being serialized